src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Microsoft.Extensions.Configuration.Binder.SourceGeneration.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(MicrosoftCodeAnalysisVersion_LatestVS)" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.7.0-3.23314.3" PrivateAssets="all" />
```

Comment:
>"4.7.0-3.23314.3" [](http://example.com/codeflow?start=81&length=17)

are we going to stick with this as a hardcoded value and not defined in the props file.

Reply:
I'll circle back to this (in this PR). I was getting various build errors from other projects when I made this global.